bigtop_toolchain/manifests/isal.pp

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+class bigtop_toolchain::isal {
+  require bigtop_toolchain::packages
+  $url = "https://github.com/intel/isa-l/archive/refs/tags/v2.29.0.tar.gz"
+  $isal = "v2.29.0.tar.gz"
+  $isaldir = "isa-l-2.29.0"
+  exec { "download isal":
+    cwd     => "/usr/src",
+    command => "/usr/bin/wget $url && mkdir -p $isaldir && /bin/tar -xvzf $isal -C $isaldir --strip-components=1",
+    creates => "/usr/src/$isaldir",
+  }
+  exec { "install isal":
+    cwd     => "/usr/src/$isaldir",
+    command => "/usr/src/$isaldir/autogen.sh && /usr/src/$isaldir/configure && /usr/bin/make && /usr/bin/make install",
```

Comment:
How about adding `--prefix=/usr/local` to avoid conflict with package like libisal provided by OS distribution (in future?).

Reply:
Okay, I'll make some changes to this section

Reply:
I just modified the configuration section to specify/usr/local and verified it on Rockylinux arrch64
![1690797165098](https://github.com/apache/bigtop/assets/24928039/c3e9b5af-963b-4ed2-9ee4-2132a22e63de)